.github/workflows/continue-docs-review.yml

```
@@ -1,114 +1,114 @@
-      - '!**/*.stories.{ts,tsx,js,jsx}'
-      - '!**/dist/**'
-      - '!**/build/**'
-  issue_comment:
-    types: [created]
-  workflow_dispatch: # Allow manual testing
-    inputs:
-      pr_number:
-        description: 'Pull request number to review'
-        required: true
-        type: string
+# name: Continue Documentation Review
```

actionlint (1.7.9)

workflow is empty

(syntax-check)